R/isomap.R

```diff
@@ -192,21 +192,22 @@ prep.step_isomap <- function(x, training, info = NULL, ...) {
 
 #' @export
 bake.step_isomap <- function(object, new_data, ...) {
-  check_new_data(names(object$columns), object, new_data)
+  col_names <- names(object$columns)
+  check_new_data(col_names, object, new_data)
 
-  if (object$num_terms > 0 && length(object$columns) > 0L) {
-    isomap_vars <- colnames(environment(object$res@apply)$indata)
```

Comment:
Poked around a bit in blames and saw that `isomap_vars` used to be used everywhere where `names(object$columns)` now is, so the distinction between the two indeed is not needed. :)